tests/unit/test_handler.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+
+from pd2jira import app
+
+@pytest.fixture()
+def triggerIssue():
+
+
+
+def test_lambda_handler(mocker):
+
+    assert true
```

Comment:
Well, at least you thought about testing

Reply:
The app depends on dynamodb to function, so even if you run it locally you still have to deploy it in "dev" to get a dynamodb to talk to. I think there's a way to run dynamo locally and want to research that.. but haven't yet.

Reply:
Wow, you can do that https://hub.docker.com/r/amazon/dynamodb-local